java/README.MD

```diff
@@ -1,24 +1,33 @@
-# Azure IoT Central ARM Java SDK samples
+# Azure IoT Central Java SDK Samples for Management APIs
 
+Sample code for using Azure IoT Central Management Plane APIs.
 Sample code for using [com.microsoft.azure.iotcentral.v2018_09_01:azure-mgmt-iotcentral](https://search.maven.org/artifact/com.microsoft.azure.iotcentral.v2018_09_01/azure-mgmt-iotcentral/1.0.0/jar) SDK provided by [Microsoft Azure](https://github.com/Azure). This sample code will check if the resource name is available, create or update an app, retrieve app information, update app, list all the apps that belongs to the specific resource group, and finally delete the app if uncommented.
 
 ## Get started
 
 ### Prerequisites
 - [Java JDK or JRE](https://www.oracle.com/technetwork/java/javase/downloads/index.html)
-- [Gradle](https://gradle.org/install/)
-- A resource group called **myResourceGroup** in your Azure subscription
+- [Maven](http://maven.apache.org/install.html)
+- A service principal and a resource group to work with. Follow [these instructions](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal).
 
-### Installation
-To begin, simply clone this repository onto your local machine and build it using Gradle.
+### SDK version
 
-```
-gradle build
-```
+The support SDK version is [1.0.0-beta.2](https://github.com/Azure/azure-sdk-for-java/tree/azure-resourcemanager-iotcentral_1.0.0-beta.2/sdk/iotcentral/azure-resourcemanager-iotcentral).
```

Comment:
will there be versions without the beta.2 tag? GA one?

Reply:
I thought a stable version had been published already, if not it seems like we missed this and it should be done.